File: romanToInteger/roToInt.py
class Solution:
    def romanToInt(self, s: 'str') -> 'int':
        arr = {'I':1,
              'V':5,
              'X':10,
              'L':50,
              'C':100,
              'D':500,
              'M':1000}
        i,total=0,0
        while(i<len(s)-1):
            if(arr[s[i]]<arr[s[i+1]]):
                total-=arr[s[i]]
            else:
                total+=arr[s[i]]
            i+=1
        total+=arr[s[-1]]
        return total
                

File: romanToInteger/roToInt.php
class Solution {

/**
 * @param String $s
 * @return Integer
 */
function romanToInt($s) {
    $arr = array("I"=>1,
                "V"=>5,
                "X"=>10,
                "L"=>50,
                "C"=>100,
                "D"=>500,
                "M"=>1000);
    $sum = 0;
    for($i=0;$i<strlen($s);$i++){
        if($arr[substr($s,$i,1)]<$arr[substr($s,$i+1,1)]){
            $sum-=$arr[substr($s,$i,1)];
        }
        else{
            $sum+=$arr[substr($s,$i,1)];
        }
    }
    return $sum;
}
}